### DISCUSSION TITLE
Modules Window: scrolling filter menus

### DISCUSSION THREAD

**Discussion by TuonyBT** [DISTRHO/Cardinal#518] [Q&A] [answered]
I am running Cardinal Main as LV2 under Carla on AV Linux. Almost everything I have tried so far is working very well, but I have one small problem with scrolling the filter menus in the modules window: I cannot scroll beyond the bottom of the window. As far as I can tell, there is no scroll bar, the cursor down and page down keys do nothing, and I cannot 'pull' down with the mouse-pointer. Do others have a similar experience or might it be peculiar to my setup?
![20230329_cardinal_menus](https://user-images.githubusercontent.com/40490096/228487544-6c5e58cc-fafb-4deb-878a-c6afdd313f41.png)

**Comment by falkTX** [comment, marked as answer]
it is a buggy behaviour imported from VCV Rack, only mousewheel works on such big menus

**Reply by TuonyBT** [reply]
Thank you very much.
